ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/policy-maps.md

```diff
@@ -108,4 +113,13 @@
                 # Range in bytes <256-128000000>.
                 higher_rate_burst_size: <int>
                 higher_rate_burst_size_unit: <str; "bytes" | "kbytes" | "mbytes" | "packets"; default="bytes">
+      copp:
+
+        # Name of control-plane poilcy.
+        name: <str>
+        classes:
```

Comment:
I tested on a few different hardware devices and we also need to add the bandwidth option:
```
[no|default] bandwidth ( kbps | pps ) BANDWIDTH
```

Reply:
Fixed.